tests/mocks.ts

```diff
@@ -567,7 +659,12 @@ export const withNullRelationsMock = {
         "slug": "ya",
         "parent": {
             "type": "category",
-            "id": "0"
+            "id": "0",
+            "slug": "home",
+            "parent": null,
+            "relationshipNames": [
+                "parent"
+            ]
```

Comment:
This change also fixes this test. The relation from category to parent should be to the object itself (including it's relation to null) and not to a copy that doesn't include this relation.